Orange/classification/neural.py

```diff
@@ -264,7 +264,7 @@ def __call__(self, example, result_type=Orange.core.GetValue):
         :rtype: :class:`Orange.data.Value`, 
               :class:`Orange.statistics.Distribution` or a tuple with both
         """
-        example = Orange.data.Instance(self.domain, example)
+        #example = Orange.data.Instance(self.domain, example)
```

Comment:
@kernc  yeah, it should be deleted I think. It was added by @ales-erjavec and I was not sure whether it is necessary in some cases. When I was using NN node to validate a test set, I found that the 'example' is a list of orange object which can't be cast into float. I don't know about the class of instance. It worked after I commented it refering to an old version. :relaxed: